src/transition/properties.js

```diff
@@ -52,7 +52,7 @@ function getTransitionProperties() {
 
   let vendorMap = {
     O: e => `o${e.toLowerCase()}`,
-    Moz: e => `moz${e}`,
+    Moz: e => e.toLowerCase(),
```

Comment:
I can't find any reference to `mozTransitionEnd`.
For instance
> FWIW, mozTransitionEnd has never been implemented in Firefox. The lowercase (per spec) transitionend is required.
https://github.com/madrobby/zepto/pull/742#issuecomment-26948989